.github/workflows/cross-test.yml

```diff
@@ -48,3 +48,43 @@ jobs:
       - uses: actions/checkout@v4
       - name: PPC64LE Build/Test
         run: tests/ci/run_cross_tests.sh ppc64le powerpc64le-unknown-linux-gnu "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_BUILD_TYPE=Release -DFIPS=1 -DBUILD_SHARED_LIBS=1"
+  riscv64-non-fips-build-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install qemu
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install qemu-user qemu-user-binfmt
+      - uses: actions/checkout@v4
+      - name: RISC-V 64 Build/Test
+# The flag below is set to avoid the following error with GCC 11.4.0:
+#
+#  /home/runner/work/aws-lc/aws-lc/crypto/pem/pem_lib.c:705:11: error: 'strncmp' of strings of length 1 and 9 and bound of 9 evaluates to nonzero [-Werror=string-compare]
+#  705 |       if (strncmp(buf, "-----END ", 9) == 0) {
+#      |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        env:
+          CFLAGS: "-Wno-string-compare"
+        run: tests/ci/run_cross_tests.sh riscv riscv64-unknown-linux-gnu "-DCMAKE_BUILD_TYPE=Release"
+  armv6-non-fips-build-test:
```

Comment:
Also this raises the question: should we have an ARM version section to the readme?

Reply:
Perhaps it could be clarified.  The earliest `arm32` platform we can support might be `armv4`.